### PR TITLE
Update custom backoffice html view

### PR DESCRIPTION
**Pull request by ault**
To preserve functionality and ensure a consistent look/feel:
* Div has been changed to Button
* Class has been added to preserve drag-to-reorder/sort
* ng-click has been updated to current default
* Comment below code added to explain importance of the class

### CODE REVIEW

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md [hunk 1]
```diff
@@ -277,12 +277,14 @@ If you'd like to display properties of `settings`, you can access these by `bloc
...
 ```html
-<div ng-controller="customBlockController" ng-click="block.edit()"> 
+<button  type="button" ng-controller="customBlockController" ng-click="api.editBlock(block, block.hideContentInOverlay, index, parentForm)"  class="btn-reset umb-outline blockelement-labelblock-editor blockelement__draggable-element ng-scope ui-sortable-handle" > 
     <h2 ng-bind="block.data.headline"></h2>
     <p ng-bind="block.data.description"></p>
-</div>
+</button>
 ```
 
+NB: the class declaration is necessary to allow the block to behave like the default blocks (i.e. drag-to-sort etc.)
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md:286]:
```suggestion
:::note
The class declaration is necessary to allow the block to behave like the default blocks (i.e. drag-to-sort etc.)
:::
```

**Review by sofietoft — COMMENTED** [review]
Thanks for suggestions the changes here @ault ! 💪 

I've added a single suggestion to use our note-formatting.

I'll make sure to get the rest reviewed as soon as possible.

**Comment by sofietoft** [comment]
Thanks for applying the suggestion @ault ! 😄 

Also, I've had the PR reviewed now, and it all checks out of course - so I'll go ahead and merge it in.
Thanks again for updating this article 💪 Much appreciate.

I can see that this is also your first-ever contribution to the UmbracoDocs - that's **awesome**! 😎 
Do you have an account on Our? 'Cause then I can give you a shiny C-trib badge for your efforts 😄 
Let me know!

**Comment by ault** [comment]
> Do you have an account on Our? 'Cause then I can give you a shiny C-trib badge for your efforts 😄

I have linked Github and Our accounts... if that doesn't help, I'm the only Ault Nathanielsz in there.  ('cause who doesn't like badges?)

**Comment by sofietoft** [comment]
Ah, you're right of course 😅 I keep forgetting about that feature!

Anyways. You have your new badge now 😄 Thanks again for helping out.
Hope to see you back here with more contribs in the future 💪

**Pull request merged**